ecies.c

```diff
@@ -117,23 +109,23 @@ int decipher(const EC_KEY *key,
         printf("\n");
 
         size_t S_len = BN_num_bytes(S);
-        unsigned char password[S_len];
```

Comment:
It's issue of the MSVC compiler - [stackoverflow link to the explanation](https://stackoverflow.com/a/312125/2626642)